templates/observability/managed.html

```diff
@@ -0,0 +1,577 @@
+          </div>
+          <div class="p-logo-section__item">
+            {{ 
+              image (
+              url="https://assets.ubuntu.com/v1/fc61db3b-Anbox-logo-azure.png",
+              alt="",
+              width="310",
+              height="310",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"}
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/5daf7029-Openstack_logo.png",
+              alt="",
+              width="292",
+              height="292",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"}
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ 
+              image (
+              url="https://assets.ubuntu.com/v1/5dfc272a-vmware_logo.png",
+              alt="",
+              width="292",
+              height="292",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"}
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ 
+              image (
+              url="https://assets.ubuntu.com/v1/f1d885ce-Oracle-cloud.png",
+              alt="",
+              width="522",
+              height="161",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"}
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ 
+              image (
+              url="https://assets.ubuntu.com/v1/53d93b1d-Anbox-logo-maas.png",
+              alt="",
+              width="310",
+              height="310",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"}
+              ) | safe
+            }}
+          </div>
+          <div class="p-logo-section__item">
+            {{ 
+              image (
+              url="https://assets.ubuntu.com/v1/dba11aee-kubernetes.svg",
+              alt="",
+              width="62",
+              height="60",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-logo-section__logo"}
+              ) | safe
+            }}
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h3>Powerful monitoring, simple and predictable price, cost-efficient</h3>
+    <p>We manage the best open source monitoring tools at both the host and the guest levels. <span class="p-heading--5">Charges are for the hosts running the monitoring tools themselves.</span> We do not charge for the systems you monitor with the tools we manage for you!</p>
```

Comment:
```suggestion
    <p>We manage the best open source monitoring tools at both the host and the guest levels. <strong>Charges are for the hosts running the monitoring tools themselves.</strong> We do not charge for the systems you monitor with the tools we manage for you!</p>
```
It might be better to use <strong> here as it wont be affected by style changes to default font-size or p-heading--five.